src/routes/docs/builders/tags-input/(sections)/Description.svelte

```diff
@@ -0,0 +1,351 @@
+<script lang="ts">
+	import { Docs } from '$routes/(components)';
+</script>
+
+<Docs.H2>Builder Description</Docs.H2>
+
+<Docs.P class="mt-4">
```

Comment:
This section is quite elaborate, and I appreciate all the effort that's gone into it! I think it's easy to get lost in it a bit though.

In general, Melt UI's philosophy for approaching docs is to first show concise usage exampoles for common use cases, and show an API reference for anything more specific. Skimming through this section, I think it can be repurposed so that certain parts are moved to Usage, and others are moved to API Reference. Wdyt?

Reply:
Otherwise, we need to think of a clear way to show information like this across builders.

Reply:
Yeah. I guess I went a little overboard with the docs.

Tags input has a lot of moving parts to it so I tried to covey how to use it clearly. 

I’ll strip it all back though

Reply:
It would be great if the api section supported a description column to add a little blurb..